app/lib/screens/HomeScreens/ChatScreen.dart

```diff
@@ -115,7 +115,7 @@ class _ChatScreenState extends State<ChatScreen> {
     final index = _messages.indexWhere((element) => element.id == message.id);
     final updatedMessage = _messages[index].copyWith(previewData: previewData);
 
-    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
Please update your flutter SDK to 3.0 as this is not the error and revert this change in new commit. This is deprecated and it isn't required to make ```WidgetBindings``` nullable now.

Reply:
okay